client/src/utils/track.ts

```diff
@@ -78,87 +76,62 @@ export const track = (event: AnalyticsEvent): Promise<unknown> | undefined => {
 };
 
 export const trackPageVisit = (): void => {
-  // @ts-ignore
   if (Build.isBrowser && typeof s != "undefined" && !firstPageOfVisit) {
-    // @ts-ignore
     s.pageURL = window.location.href;
-    // @ts-ignore
     s.t();
   }
   firstPageOfVisit = false;
 };
 
 export const trackPageFetchException = (): void => {
-  // @ts-ignore
   if (Build.isBrowser && typeof s != "undefined") {
-    // @ts-ignore
     s.linkTrackVars =
       "prop39,prop41,prop50,prop61,prop62,eVar39,eVar41,eVar50,eVar61,eVar62,eVar69";
-    // @ts-ignore
     s.tl(true, "o", "page fetch exception");
   }
 };
 
 export const trackExternalLink = (hrefTo: string): void => {
-  // @ts-ignore
   if (Build.isBrowser && typeof s != "undefined") {
-    // @ts-ignore
     s.linkTrackVars =
       "prop39,prop41,prop50,prop61,prop62,eVar39,eVar41,eVar50,eVar61,eVar62,eVar69";
-    // @ts-ignore
     s.tl(true, "e", hrefTo);
   }
 };
 
 export const setSearchQuery = (query: string): void => {
-  // @ts-ignore
   if (Build.isBrowser && typeof s != "undefined") {
-    // @ts-ignore
     s.eVar26 = query;
   }
 };
 
 const triggerNoSearchResults = (query: string): void => {
-  // @ts-ignore
   const queryBackup: string = s.eVar26;
-  // @ts-ignore
-  const resultCountBackup: number = s.eVar27;
+  const resultCountBackup: number = parseInt(s.eVar27);
```

Comment:
**Suggestion**:

What do you think about passing `10` as the second argument here to be sure the string is always being parsed as a base 10 number?  From MDN, [`10` is not always the default value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#parameters). My guess is that the default radix depends on the browser vendor implementation, so I think it would be safer to be explicit here.

```suggestion
  const resultCountBackup: number = parseInt(s.eVar27, 10);
```

Reply:
Yeah, I totally forgot about that!